src/main/resources/application.conf

```diff
@@ -0,0 +1,15 @@
+akka {
+  event-handlers = ["akka.event.slf4j.Slf4jEventHandler"]
+  loggers = ["akka.event.slf4j.Slf4jLogger"]
+  loglevel = "DEBUG"
+  actor {
+    debug {
+      //autoreceive = on
+      //lifecycle = on
+      receive = on
+      event-stream = on
+      fsm = on
+      //unhandled = on
```

Comment:
I'll just remove them, it's mostly just an artifact of figuring out how the logging works